rails-commenting.rb

```diff
@@ -9,22 +9,22 @@
 # ---1)
 class BlogPostsController < ApplicationController
   def index
-    # ---2)
+    # ---2) .all will display all of the blog posts.
     @posts = BlogPost.all
   end
 
   def show
-    # ---3)
+    # ---3) .find will find a blog post based on the i.d. of the blog post.
     @post = BlogPost.find(params[:id])
   end
 
-  # ---4)
+  # ---4) new will return a new blog post.
   def new
     @post = BlogPost.new
   end
 
   def create
-    # ---5)
+    # ---5) .create will create a new blog post
```

Comment:
Good! The instance variable @post is allowing that to happen with some strong params